frontend/components/SwitchHeader/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withRoute } from '@shopgate/engage/core';
+import getConfig from '../../helpers/getConfig';
```

Comment:
i always wondered why is see this `getConfig` helper so often in PS extensions, but i was afraid to ask.

i don't really know about real live situations where the config.json isn't available right from the beginning. this approach just introduces an line of code with a function call.

```js
import getConfig from '../../helpers/getConfig';
const { pageLinking } = getConfig();
```
vs direct import

```js
import { pageLinking } from '../../config';
```

Reply:
I changed all function calls to the direct import